src/client/_clientwindow.py

```diff
@@ -870,12 +874,18 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def on_actionSetAutoJoinChannels(self):
+        input, ok = QInputDialog.getText(None, 'Set autojoin channels','Previous list : '+str(Settings.get('chat/auto_join_channels'))+'\nEnter the list of channels you want to join, separated by ;\n\n For example: #poker;#newbie\nTo disable autojoin, leave box empty and press ok', QLineEdit.Normal, "")
```

Comment:
Setting message box text in a variable first would be a bit more readable - especially since we could use multiple lines for that (after each '\n'). Instead of showing the previous list in the text we can just set dialog box contents.

Also 'input' is a python global function, please rename it.

Reply:
Done, maybe not sure how you wanted to do it.